libkbfs/bare_root_metadata_v3_test.go

```diff
@@ -15,12 +15,13 @@ import (
 )
 
 func TestBareRootMetadataVersionV3(t *testing.T) {
-	tlfID := tlf.FakeID(1, false)
+	tlfID := tlf.FakeID(1, tlf.Private)
```

Comment:
I can't easily add any tests until the next PR (#996) which vendors in `keybase1.MakeTestTeamID()`.  I'll add a simple bare RMD test to that one.  Later PRs have more involved libkbfs tests.

As for `test/`, that will require some DSL changes.  I'll do those in a separate PR.